operator/manifests/default-istio-operator-cluster.yaml

```
@@ -0,0 +1,319 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  name: installed-state-default-operator
```

Istio installation adds `installed-state-` prefix
```suggestion
  name: default-operator
```